transparency-log.md

```diff
@@ -0,0 +1,51 @@
+# US-RSE Code of Conduct Transparency Log
+
+The US-RSE expects everyone who participates in the community will honor our
```

Comment:
follow instead of honor?

Reply:
This was used to match the working the in the [Code of Conduct](https://us-rse.org/about/code-of-conduct/) itself (Introduction section).